src/seeds.proposals.cpp

```diff
@@ -528,8 +528,13 @@ void proposals::evalproposal (uint64_t proposal_id, uint64_t prop_cycle) {
       }
 
     } else {
+
+      asset stackToBurn = valid_quorum ?  pitr->staked : asset(pitr->staked.amount * 0.05, seeds_symbol);
+      stackToBurn = passed ? stackToBurn : pitr->staked;
```

Comment:
This is a little weird - an explicit if statement that defines all possible cases would be easier to understand. 

But - it's correct code.